index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Capstone Project</title>
+</head>
+<body>
+    <!--Header Section-->
+    <section id="header__section" class="header__section">
+        <header class="header__container">
+            <a href="#header__section" class="logo__image">
+                <img src="./images/logo.png" alt="logo_image">
+            </a>
+            <nav class="header__navbar">
+                <i id="menuOpen" class="fa-solid fa-bars"></i>
+                <i id="menuClose" class="fa-solid fa-times"></i>
+                <ul class="navbar__container">
+                    <li class="navbar__list__item ">
+                        <a href="" class="navbar__list__link active">About</a>
+                    </li>
+                    <li class="navbar__list__item">
+                        <a href="" class="navbar__list__link">Program</a>
+                    </li>
+                    <li class="navbar__list__item">
+                        <a href="" class="navbar__list__link">Join</a>
+                    </li>
+                    <li class="navbar__list__item">
+                        <a href="" class="navbar__list__link">Sponsor</a>
+                    </li>
+                    <li class="navbar__list__item">
+                        <a href="" class="navbar__list__link">News</a>
+                    </li>
+                    <li class="navbar__list__item campaign__list">
+                        <a href="" class="navbar__list__link campaign__link">CC Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+    </section>
+    <!--End-->
+
+    <!--Main Section-->
+    <main class="main__section">
+        <section class="main__container">
+            <figure>
+                <img class="main__image" src="./images/title.png" alt="title_image">
+            </figure>
+            <article class="main__features">
+                <p class="main__paragraph">
+                    A joyful celebration believing in the value of opennes and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.
+                </p>
+                <h1 class="main__title">2015.10.15(THU) - 16(FRI)</h1>
+                <h2 class="main__subtitle">@ National Museum of Korea, Art Center Nabi and more</h2>
+            </article>
+        </section>
+    </main>
```

Comment:
According to the project requirements, Kindly I suggest you personalize the content i.e., instead of a conference you can build a website for a concert or for a web development course. 😄 

><img width="1440" alt="Screenshot 2023-06-28 at 4 33 39 PM" src="https://github.com/Feliperto/Capstone-Project/assets/91237525/12c37023-896e-4785-8c7d-6b9f411ca0fc">